auth/src/main/java/com/firebase/ui/auth/ui/email/EmailActivity.java

```diff
@@ -123,7 +124,11 @@ public void onNewUser(User user) {
             FragmentTransaction ft = getSupportFragmentManager().beginTransaction()
                     .replace(R.id.fragment_register_email, fragment, RegisterEmailFragment.TAG);
 
-            if (emailLayout != null) ft.addSharedElement(emailLayout, "email_field");
+            if (emailLayout != null) {
+                String emailFieldName = getString(R.string.fui_email_field_name);
+                ViewCompat.setTransitionName(emailLayout, emailFieldName);
+                ft.addSharedElement(emailLayout, emailFieldName);
```

Comment:
@clydebarrow Thanks again for the PR! I've tested your fix and it turns out view compat simply no-ops on API 19 and below. To make that clearer in _our_ code, it would be better to simply not add the transition on API 19 and below:
```java
if (Build.VERSION.SDK_INT > Build.VERSION_CODES.KITKAT && emailLayout != null) {
    ft.addSharedElement(emailLayout, getString(R.string.fui_email_field_name));
}
```

Reply:
According to the information I have, the support library (which is used here) provides the transition features back to API 14. I'm attempting to verify that it makes a visual difference on API 19 right now, but have been sidetracked by another issue.

Reply:
Yeah, I didn't notice a transition even with the view compat thing.

Reply:
Also, I couldn't find docs on the API 14 fragment transactions, would you mind sharing?

Reply:
The release notes for the support library 27.0.0 are here:
https://developer.android.com/topic/libraries/support-library/revisions.html#27-0-0

and say inter-alia "Fragment can use support library versions of Transition for fragment transitions, including shared-element transitions."

The support libraries have a minimum API level of 14.

I have confirmed that the transition on API 19 is present (at least it looks identical visually to running on API 25) so I suggest keeping the code as per my PR.

Reply:
SGTM, @samtstern should be back soon with actual push access.